### DISCUSSION TITLE
Salon Topic 7. Building a Secure Infrastructure Ecosystem

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#94] [Silicon Salon]
Finally, tell us your thoughts on how we can go from our individual interests to a fully secure ecosystem.

> Ultimately, we have to ask who we're missing from our discussions, and how we can bring them in, to improve the cross-discussion of requirements, and ultimately the interoperability of our field. We'd like to see more cryptographers and to have more presentations by a variety of people next time we host a Silicon Salon.

> There were also interesting discussions about whether we can cooperate as a field, trading peer review with each other, for example, to create a more robust ecosystem.

**Comment by kanzure** [comment]
Curious to know if Assurance Contracts ( https://en.wikipedia.org/wiki/Assurance_contract ) could help the ecosystem to organize around standard interfaces for hardware wallets.
